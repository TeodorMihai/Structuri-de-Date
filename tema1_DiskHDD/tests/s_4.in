2 8
::w 1 0 3000
3
::w 2 0 2000
1
::w 3 0 1000
3
::d 2 0
5
::r 3 0
1
::d 2 0
5
::d 3 0
1
::w 7 511
1
::d 7 511
800
::d 7 511
400
::w 7 2046 AAAA
20000
::w 0 0 ABCD
5
::w 0 0 0ABC
5
::r 0 0
2
::r 0 0
2
::r 0 15
16
::d 0 0
1
::w 0 15 AAAA
15
::r 0 15
1
::e