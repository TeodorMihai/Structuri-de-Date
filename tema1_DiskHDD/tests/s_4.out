2
40
1000
41
1
318
0ABC
ABCD
0000
1.41 0.50 0.50 18.64
